src/main/java/controller/AddArticleServlet.java

```diff
@@ -0,0 +1,34 @@
+package controller;
+
+import dao.DataBase;
+import model.Article;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet(value = "/addArticle")
+public class AddArticleServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("addArticle.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String title = req.getParameter("title");
```

Comment:
better add checking if parameters is empty

Reply:
fixed